integration-tests/features/leapp_testing/__init__.py

```diff
@@ -185,7 +187,8 @@ def redeploy_as_macrocontainer(self, source_vm, target_vm):
 
     def check_response_time(self, cmd_args, time_limit, *,
                             complete_user=False,
-                            complete_identity=False):
+                            complete_identity=False,
+                            as_sudo=False):
```

Comment:
#120 also makes changes to the parameters here, so these PRs are going to conflict